src/schemas/21-526EZ/schema.js

```diff
@@ -373,80 +327,7 @@ let schema = {
       required: ['servicePeriods', 'servedInCombatZone']
     },
     disabilities: {
```

Comment:
If this property has the same name as the definition, is it necessary?

Reply:
You mean the `$ref`? Yeah, I dunno to be honest. Looks like this is what we do in some of the other forms, e.g. 527EZ (`additionalSources: { $ref: '#/definitions/additionalSources' }`). It's probably more consistent with JSON schema style. 🤷‍♂️

Reply:
Yeah, I think we still need the property even if we define the definition.